fhireval/test_suite/bulk_export/__init__.py

```diff
@@ -1,6 +1,6 @@
 
 # The bullet points associated with this set of tests in the evaluation document
-test_set_id = "2.6"
+test_set_id = "2.09"
```

Comment:
Is this correct? Maybe either`"2.0.9"` or `"2.9"`?

Reply:
Yeah, it's supposed to be that way. Changed to ensure that the tests are run in the correct order.